src/osdc/Objecter.cc

```diff
@@ -2933,10 +2923,21 @@ void Objecter::_session_op_assign(OSDSession *to, Op *op)
   op->session = to;
   to->ops[op->tid] = op;
 
+  if(op->on_noosd_timeout && (! to->is_homeless())){
```

Comment:
add space after `if`, add remove the space after `!`. and please drop the parenthesis around `! to->is_homeless()`.

Reply:
Done